cpp/inc/bond/core/value.h

```diff
@@ -605,6 +633,24 @@ class value<void, Reader>
 };
 
 
+template <typename Protocols, typename Transform>
+void DeserializeContainer(const Transform& transform, const value<void, detail::SchemaReader&>& element, detail::SchemaReader& input)
+{
+    switch (element.GetTypeId())
+    {
+    case bond::BT_SET:
```

Comment:
what do you think by putting these cases into function to reduce code duplication?

Reply:
This way it reads identical as with other similar checks here.